action.yml

```diff
@@ -0,0 +1,168 @@
+name: Code Engine GitHub Action
+description: Github action to interact with IBM Cloud Code Engine deploy Apps, Jobs and Functions
+author: IBM
+
+branding:
+  icon: cloud
+  color: blue
+
+inputs:
+  api-key:
+    description: IAM API Key used to log into the IBM Cloud
+    required: true
+
+  resource-group:
+    description: An IBM Cloud Resource Group is a logical container used to organize and manage related cloud resources
+    required: false
+
+  region:
+    description: The Region refers to a geographical area where the project is are located
+    required: true
+
+  project:
+    description: A Code Engine Project Is the grouping of your Apps, Functions and Jobs
+    required: true
+
+# App, Function or Job Specific inputs
+  entity:
```

Comment:
We should rename `entity` to `component`

Reply:
Added the changes in the readme